### DISCUSSION TITLE
How can I retrain existing spacy v3 NER model with additional data?

### DISCUSSION THREAD

**Discussion by laxmimerit** [explosion/spaCy#8513] [Help: Coding & Implementations] [answered]
Hi,
I have trained Spacy v3 model on 1 million rows and want to train for additional 1 million.
```
!python -m spacy train spacy_v3/config.cfg --output spacy_v3/output --paths.train spacy_v3/train.spacy --paths.dev spacy_v3/test.spacy --gpu-id 0
```
While training on additional 1 million or more data, I want to start my training from the previous best-model. Is there anyway to retrain existing ner model?

**Comment by polm** [comment, marked as answer]
You can resume training a model, see #8176 for a guide. However, if you train on a completely different set of training data you are likely to trigger catastrophic forgetting, which means that your model will forget everything not in the current training data. See the FAQ in #8226 for some links explaining what that is and how to work around it, though generally the best thing is to just train on all your training data at once.
